frontend/src/pages/Contact.jsx

```
@@ -0,0 +1,368 @@
+              }`}
+            >
+              Welcome
+            </button>
+            <button 
+              onClick={() => setActiveSection('contact')}
+              className={`px-8 py-3 rounded-full text-lg font-medium transition-all duration-300 cursor-pointer ${
+                activeSection === 'contact' 
+                  ? 'bg-blue-600 text-white shadow-lg' 
+                  : 'bg-transparent text-gray-600 hover:bg-gray-100'
+              }`}
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)